examples/voting/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16-slim AS build
+WORKDIR /build
+COPY package.json package-lock.json ./
+RUN npm install
+COPY . .
+RUN npm run build
```

Comment:
Is there a reason we use npm here, instead of yarn like we do elsewhere.
(I guess maybe it was because of the create-react-app stuff)
(I haven't tested that this works after my changes :upside_down_face: )

But I would assume we could use basically the same dockerfile as wccd's here. :thinking: